.github/dependabot.yml

```
@@ -19,3 +19,11 @@ updates:
   - dependency-name: lxml
     versions:
     - 4.6.2
+- package-ecosystem: github-actions
+  directory: /
```

Match the one above:
```suggestion
  directory: "/"
```